dts: bindings: Fix missing "zephyr" vendor name

yboutreux opened the pull request:
Fix missing "zephyr" vendor name, currently used with `zephyr,sim-flash`, `zephyr,bt-hci-spi` and `zephyr,bt-hci-spi-slave`.
This avoid checkpatch warning : _DT compatible string vendor "zephyr" appears un-documented_
    
Signed-off-by: Yaël Boutreux <yael.boutreux@st.com>

galak commented on dts/bindings/vendor-prefixes.txt:
```diff
@@ -456,6 +456,7 @@ xunlong	Shenzhen Xunlong Software CO.,Limited
 ysoft	Y Soft Corporation a.s.
 zarlink	Zarlink Semiconductor
 zeitec	ZEITEC Semiconductor Co., LTD.
+zephyr	Zephyr
```
Can you make this:

`zephyr zephyr-specific binding`

yboutreux replied:
Sure! This has been updated

galak reviewed (changes requested):
Minor change

yboutreux committed:
dts: bindings: Fix missing "zephyr" vendor name

Fix missing "zephyr" vendor name, currently used with
zephyr,sim-flash; zephyr,bt-hci-spi and zephyr,bt-hci-spi-slave.
This avoid checkpatch warning :
"DT compatible string vendor "zephyr" appears un-documented".

Signed-off-by: Yaël Boutreux <yael.boutreux@st.com>
